addons/im_livechat/static/src/legacy/widgets/public_livechat_view/public_livechat_view.xml

```diff
@@ -173,11 +173,6 @@
                                 t-attf-class="o_thread_author o_PublicLivechatMessage_headerAuthor #{message.shouldRedirectToAuthor() ? 'o_mail_redirect' : ''}">
                             <t t-esc="message.getDisplayedAuthor()"/>
                         </strong>
-                        <strong t-elif="message.hasEmailFrom()">
-                            <a class="text-muted" t-attf-href="mailto:#{message.getEmailFrom()}?subject=Re: ">
-                                <t t-esc="message.getEmailFrom()"/>
-                            </a>
-                        </strong>
```

Comment:
For some reasons, `getEmailFrom` was already removed... hence why no removal of this obsolete method in the diff of this PR